chains/substrate/listener.go

```diff
@@ -163,13 +167,15 @@ func (l *listener) pollBlocks() error {
 				l.log.Error("Failed to write to blockstore", "err", err)
 			}
 
+			if l.metrics != nil {
+				l.metrics.BlocksProcessed.Inc()
```

Comment:
Just a Q, if we were to need to reprocess blocks for some reason this counter could theoretically be larger than the latest processed block metric right?

Reply:
Yup! The blocks processed counter will also reset to 0 on restarts.

Reply:
do you think it might make more sense to specify that this is the blocks processed in a given "session" then in the docs?

Reply:
Not really, this is almost always the case with counters such as this